redbot/cogs/customcom/customcom.py

```diff
@@ -204,15 +204,16 @@ def __init__(self, bot):
         """Custom commands management."""
         pass
 
-    @customcom.group(name="create", aliases=["add"])
+    @customcom.group(name="create", aliases=["add"], invoke_without_command=True)
     @checks.mod_or_permissions(administrator=True)
-    async def cc_create(self, ctx: commands.Context):
+    async def cc_create(self, ctx: commands.Context, command: str.lower, *, text: str):
         """Create custom commands.
 
+        If a type is not specified, a simple CC will be created.
         CCs can be enhanced with arguments, see the guide
         [here](https://red-discordbot.readthedocs.io/en/v3-develop/cog_customcom.html).
         """
-        pass
```

Comment:
Needs additional logic for invoking help now.

Reply:
From my testing the help works fine. The group command is invoked when no subcommand could be found AND has required args, so proper syntax isn't used it sends the help message. If you have a specific instance that doesn't work, I can take a look at that.

Reply:
You're right, I read through too quickly and didn't notice the required arg.